RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -340,6 +341,13 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
      fitParsVec.push_back(0.);
      fitParsVec.push_back(999.);
      fitParsVec.push_back(false);
+   } else {
```

Comment:
@slava77 This extra else was in practice done inside the producer where the size of the returned vector was checked.
 https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/RecoLocalCalo/HcalRecAlgos/src/HcalSimpleRecAlgo.cc#L366-L369

The initial parameters I pick (time = -9999 and ampl = 0)  are basically this one
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/RecoLocalCalo/HcalRecAlgos/src/HcalSimpleRecAlgo.cc#L322-L326

For now I just restored the old behavior, there is no need to single out this case.
I prefer to fix this workflow crash now. I can came back to this later, but prefer to have new feature in later PR.